examples/tutorials.jl

```diff
@@ -35,6 +35,30 @@
         lines!(scene, x, y2, color = :black)
         scatter!(scene, x, y2, color = :green, marker = :utriangle, markersize = 0.1)
     end
+    
+    @cell "Tutorial removing from a scene" [tutorial, line, scene, markers] begin
+        x = range(0, stop = 2pi, length = 80)
+        f1(x) = sin.(x)
+        f2(x) = exp.(-x) .* cos.(2pi*x)
+        y1 = f1(x)
+        y2 = f2(x)
+
+        scene = lines(x, y1, color = :blue)
+        scatter!(scene, x, y1, color = :red, markersize = 0.1)
+
+        lines!(scene, x, y2, color = :black)
+        scatter!(scene, x, y2, color = :green, marker = :utriangle, markersize = 0.1)
+
+        display(scene)
+
+        sleep(10)
```

Comment:
Is there a need for this?  Consider using a Stepper instead?

Reply:
I like `sleep()` because you can really see everything happen before you in an interactive fashion. Some of the magic may be lost if we switch to a stepper imo.

Reply:
Your call though. I can switch it if you think a stepper will better showcase what is being done in this example.

Reply:
Hm, examples aren't supposed to work that way...
Either one executes them one by line in e.g. Atom, to follow the example, or one looks at it in the docs... And for the docs, to have the images for each step, we need the stepper ;)

Reply:
ah, well, the images are the important part so I will switch this to a stepper!

Reply:
Should be all set now!